code/modules/admin/topic.dm

```diff
@@ -934,6 +935,8 @@
 					joblist += jobPos
 			if("ghostroles")
 				joblist += list("pAI", "posibrain", "drone", "deathsquad", "lavaland")
+			if("teamantags")
+				joblist += list("operative", "revolutionary", "cultist", "servant of Ratvar", "abductor", "alien candidate")

```

Comment:
Maybe a conversion ban aswell, that would include revs, clock, cult and alien?

Reply:
Yes do that too